libraries/libfc/src/log/dmlog_appender.cpp

```diff
@@ -24,10 +24,14 @@ namespace fc {
 
    dmlog_appender::dmlog_appender( const std::optional<dmlog_appender::config>& args )
    :dmlog_appender(){
-      if (!args || args->file == "-")
+      if (!args || (args->file == "-" || args->file == "-stdout"))
```

Comment:
detail... I think the extra parenthesis are unneeded.

Reply:
Not needed, but I think it reads better.